fastlane/Fastfile

```diff
@@ -14,7 +14,7 @@ lane :release do
   #       app_identifier: "com.duckduckgo.mobile.ios",
   #       version: "7"
   #     ) + 1
-  buildNumber = 895
+  buildNumber = 896
```

Comment:
Are you bumping the minor version too?

Optional: Since this is no longer automated why not delete it from here and update it in the General panel in XCode (or Info.plist) directly? Saves the confusion of having a duplicate.

Reply:
Thanks @subsymbolic - Good spot - I thought I had done that and pushed it.

Moving forward I will probably reset the build number to 1 and then not bother with it unless we are iterating over a release. I think we discussed something like this before.  However, it's easier to bump the build number via Fastlane as Fastlane will update any extensions, etc as well.  Though since we don't really have any extensions, I guess that's kind of moot.